frontend/src/components/set-prices/SetPrices.jsx

```diff
@@ -0,0 +1,122 @@
+import React, {useEffect, useState} from "react";
+import {Formik, Form, Field, ErrorMessage} from "formik";
+import * as Yup from "yup";
+import "./SetPrices.css";
+import Button from "components/button/Button";
+
+function SetPrices(props) {
+  const [errorMessage, setErrorMessage] = useState("");
```

Comment:
`setErrorMessage` is never used, so `errorMessage` is always a blank string, why do we need it?